source/crud/tests/v2/db-aggregate.yml

```diff
@@ -1,6 +1,7 @@
 runOn:
     -
         minServerVersion: "3.6.0"
+        serverless: "forbid"
```

Comment:
I think the only options for db-level aggregate are `$listLocalSessions` and `$currentOp`, both of which are unsupported by the proxy.

And yeah, I believe whatever restrictions apply to the serverless proxy probably also apply to the free-tier proxy, since I think the former is a descendant of / shares a lot of code with the latter, not sure about how tightly coupled they are though.